pages/en/network-transition-faq.mdx

```diff
@@ -0,0 +1,236 @@
+
+Subgraphs with these features can be published to the network, but they may not be picked up by Indexers. However, subgraphs with sufficient signal may still attract Indexers interested in collecting query fees, which any subgraph is eligible for.
+
+## How much does The Graph Network cost in comparison to running my own infrastructure?
+
+The Graph's decentralized network is 60-90% less expensive than running dedicated infrastructure, as shown in [these case studies](https://thegraph.com/docs/en/network/benefits/#low-volume-user-less-than-30-000-queries-per-month).
+
+## Is there anything I should do with my hosted service subgraph after I migrate to the network?
+
+Hiding your hosted service subgraph is strongly recommended to avoid confusion. [This video](https://www.loom.com/share/7cffd2a7845e4fbd8c51f45c516cb7f9) walks through the process.
+
+## When will the decentralized network support my preferred chain?
+
+There is no set timeline per chain, they will be dictated by Indexer readiness via the [MIPs program](https://thegraph.com/migration-incentive-program/) where new chains are tested by Indexers. As new chains are supported on the network, users will receive ample notification to prepare for migration. Core devs and contributors to The Graph ecosystem are working to implement support for more chains as soon as possible.
+
+## Is Ethereum mainnet entering Phase 1 of the network transition process?
+
+While Ethereum was initially anticipated to begin transition off of the hosted service by the end of Q3 2022, this has been [postponed](https://thegraph.com/blog/transitioning-to-decentralized-graph-network) to address user feedback. Additional improvements to user experience, billing, and other fulfillments of user requests will drive Ethereum's hosted service transition timeline. Stay up to date on when Ethereum will enter The Sunray phase via the integration status tracker below and via [The Graph Twitter.](http://www.twitter.com/graphprotocol)
+
+## The Graph Network integration status tracker
+
+The table below illustrates where each chain is in the network integration process. If your preferred chain is not yet listed, integration has not yet begun, and that chain is still fully supported by The Graph's hosted service.
+
+| Chain / Network | Announcing integration on The Graph Network | Network Integration complete | Phase 1: no new subgraphs on hosted | Phase 2: disabled subgraph upgrades on hosted | Phase 3: disable subgraphs on hosted |
+| --- | --- | --- | --- | --- | --- |
+| Ethereum | ✓ | ✓ |  |  |  |
+| Gnosis (formerly xDAI) | ✓ |  |  |  |  |
+
+\*This table will not include test chains, which remain free in [Subgraph Studio](https://thegraph.com/studio/).
+
+# Query Fees, API Keys, and Billing Questions
```

Comment:
@MichaelMacaulay – Same here.

Reply:
Got all the H1s changed to H2 and all the H2s down to H3